src/lib.rs

```diff
@@ -85,6 +89,7 @@ struct LightData {
 #[derive(Debug)]
 enum SubNode {
     Empty,
+    UIText(TextData),
```

Comment:
This should be called `UiText`.

Reply:
UI is abrreviation, but I don't mind.

Reply:
This is recommended by the [Rust API guidelines](https://rust-lang-nursery.github.io/api-guidelines/naming.html) - see the note at the bottom of the table.